lib/babe/epoch.go

```diff
@@ -11,154 +11,140 @@ import (
 	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
 )
 
+var ErrEpochLowerThanExpected = errors.New("epoch lower than expected")
+
+type EpochDescriptor struct {
```

Comment:
does this need to be exported? Doesn't look like it needs to be based on checking references.  Will need to add comment if you want it to be exported.